views/index.erb

```diff
@@ -74,6 +75,9 @@
               <br>
               <input type="checkbox" name="ps_remember" style="text-decoration: none !important;"></input>
             </div>
+            <div class="col-md-3" style="margin-top: 1vh;">
+              <div style="color: #aeb1b7;">(Your account is not stored in our servers!)</div>
```

Comment:
`Your account information is not stored on our servers`

Please check how this looks on mobile -- overlaps on top of the checkbox.